db/migrate/20161115171029_add_encrypted_pii_recovery_code.rb

```diff
@@ -0,0 +1,5 @@
+class AddEncryptedPiiRecoveryCode < ActiveRecord::Migration
+  def change
+    add_column :profiles, :encrypted_pii_recovery, :text
```

Comment:
Right, so should the migration match what we're adding? Either way not a big deal